docs/design_decisions.md

```diff
@@ -212,11 +212,35 @@ Alle properties in de response worden in de Open API Specificaties gedefinieerd
 _**Ratio**_
 
 De hoeveelheid businesslogica in interface beperken. Zorgen dat zoveel mogelijk antwoord gegeven kan worden, ook wanneer een verwachte property geen waarde heeft. Het alternatief, het opnemen van de reden van geen waarde (zoals StUF:noValue) is dan niet nodig, wat het gebruik van de API eenvoudiger maakt.
-Tevens compliceert het verplicht maken van elementen de toepassing van de "fields" en de "expand" parameters en het filteren van de terug te geven gegevens op autorisatie van de organisatie. 
+Tevens compliceert het verplicht maken van elementen de toepassing van de "fields" en de "expand" parameters en het filteren van de terug te geven gegevens op autorisatie van de organisatie.
 
 ### DD5.8 Actuele zoekresultaten worden niet gesorteerd
 De API standaard schrijft niet voor hoe zoekresultaten in de API moeten worden gesorteerd. Wanneer de client behoefte heeft aan gesorteerde resultaten, moet zij de ontvangen resultaten zelf sorteren.
 Dit betekent dat er in de berichtspecificaties geen gebruik gemaakt wordt van de _'sorteer'_ parameter.
 
 ### DD5.9 Properties die gebruik maken van Booleans worden alleen geretourneerd als de waarde 'true' is
 In diverse situaties worden booleans opgenomen als er sprake is van indicatoren. Deze booleans worden alleen geretourneerd als de waarde van de boolean ook informatief is. Dit soort properties worden dus alleen opgenomen als de waarde van de Boolean 'true' is.
+
+### DD5.10 Alleen HTTP-foutcodes die kunnen voorkomen worden opgenomen in de specificatie
+Bij verschillende operaties zijn verschillende foutcodes van toepassing. Zo zal bij het bevragen van een collectie geen 404 optreden als er geen resultaat is, maar wordt er een lege collectie geretourneerd. Dan hoeft de 404 foutcode ook niet gespecificeerd te worden. Als "best practice" worden de volgend foutcodes bij per operatie gedefinieerd.
+* Bij een **get** operatie met query-parameters die een array als response oplevert:
+  * 200 Geslaagd
+  * 400 Bad request
+  * 401 Unauthorized
+  * 403 Forbidden
+  * 406 Not Acceptable
+  * 412 Precondition Failed
```

Comment:
Deze ook alleen als Crs content negotiation wordt toegepast (Accept-Crs of Content-Crs request headers).